x-pack/plugins/apm/public/components/app/Main/route_config/index.tsx

```diff
@@ -335,6 +337,7 @@ export const routes: APMRouteDefinition[] = [
     component: TraceLink,
     breadcrumb: null,
   },
+  // !! Need to be kept in sync with the searchDeepLinks in x-pack/plugins/apm/public/plugin.ts
```

Comment:
@dgieselaar The improvements to the client side routes should come in handy here https://github.com/elastic/kibana/issues/51963.

Reply:
Actually, extracting the necessary routes might be even simpler